addons/mail/models/mail_message.py

```diff
@@ -846,6 +846,17 @@ def toggle_message_starred(self):
             "mail.message/toggle_star", {"message_ids": [self.id], "starred": starred}
         )
 
+    def _send_bus_reaction_group(self, content):
```

Comment:
I would keep similar prefix as other methods for consistency: `_bus_send_...`

Reply:
And maybe move after `_message_reaction` (will it keep some parts of the diff as existing code?). Anyway it makes more sense to see `_message_reaction` before.